dashboard/README.md

```diff
@@ -1,198 +1,161 @@
-    │   │   │   ├── NavbarDrawerComponent
-    │   │   │   │   └── index.jsx
-    │   │   │   ├── OverviewComponent
-    │   │   │   │   ├── common-component.jsx
-    │   │   │   │   ├── index.jsx
-    │   │   │   │   ├── index.less
-    │   │   │   │   └── NewRunsComponent.jsx
-    │   │   │   ├── PaginationComponent
-    │   │   │   │   └── index.jsx
-    │   │   │   ├── ProfileComponent
-    │   │   │   │   ├── index.jsx
-    │   │   │   │   └── index.less
-    │   │   │   ├── SidebarComponent
-    │   │   │   │   └── index.jsx
-    │   │   │   ├── TableComponent
-    │   │   │   │   ├── common-components.jsx
-    │   │   │   │   ├── index.jsx
-    │   │   │   │   └── index.less
-    │   │   │   ├── TableOfContent
-    │   │   │   │   ├── common-components.jsx
-    │   │   │   │   ├── index.jsx
-    │   │   │   │   └── index.less
-    │   │   │   └── ToastNotificationComponent
-    │   │   │       └── index.jsx
-    │   │   └── containers
-    │   │       ├── Dashboard
-    │   │       │   └── index.jsx
-    │   │       └── MainLayout
-    │   │           ├── index.jsx
-    │   │           └── index.less
-    │   ├── reducers
-    │   │   ├── authReducer.js
-    │   │   ├── datasetListReducer.js
-    │   │   ├── endpointReducer.js
-    │   │   ├── index.js
-    │   │   ├── loadingReducer.js
-    │   │   ├── navbarReducer.js
-    │   │   ├── overviewReducer.js
-    │   │   ├── tableOfContentReducer.js
-    │   │   ├── toastReducer.js
-    │   │   └── userProfileReducer.js
-    │   ├── store
-    │   │   └── store.js
-    │   ├── utils
-    │   │   ├── axiosInstance.js
-    │   │   ├── dateFunctions.js
-    │   │   ├── filterDataset.js
-    │   │   ├── helper.js
-    │   │   └── routeConstants.js
-    │   ├── App.css
-    │   ├── App.js
-    │   ├── App.test.js
-    │   ├── index.css
-    │   ├── index.js
-    │   ├── logo.svg
-    │   ├── reportWebVitals.js
-    │   └── setupTests.js
-    ├── config-overrides.js
-    ├── jsconfig.json
-    ├── jsdoc.json
-    ├── package.json
-    ├── package-lock.json
-    └── README.md
+ .
+ ├── public									# HTML entry
+ ├── server 								# endpoint configuration
+ └── src
+     ├── actions							# redux actions
+     ├── assets
+     │   ├── constants						# constants used in code
+     │   ├── images							# images
+     │   └── logo							# Pbench logos
+     ├── modules
+     │   ├── components						# functional components
+     │   └── containers						# parent components
+     ├── reducers							# redux reducer
+     ├── store								# redux store
+     └── utils								# utility functions
 ```
 
+  
+
 ## Assets
 
+  
+
 Assets placed in the `src/assets/images` directory are only referenced within component or layout definitions and are packaged in the generated `***.js` file during the build process.
 
+  
+
 ## Prerequisites
 
+  
+
 ## Install Node JS
-Refer to https://nodejs.org/en/ to install nodejs
+
+Refer [https://nodejs.org/en/] (Node.js) official website
```

Comment:
Interestingly, I hadn't thought of this as a "placeholder" link, but as a URL link (with redundant explicit brackets) followed by a parenthesized explanation. While there may be some value to having the URL plainly visible, it appears that Markdown doesn't actually support the `[link]` syntax, and this is rendered as `[<active link>]`, which I don't think is what we want.

If we think we want to be sure the reader sees "https://nodejs.org/" (I could argue that the "/en" is unnecessary and possibly undesirable since it forces a locale/language choice on the reader, presuming that the "unencumbered" link either goes by the user's browser configuration or presents a choice), then just `https://nodejs.org` without any Markdown is good.

But if we want to do this as Webb infers, with "Node.js" as the link text redirecting to "https://nodejs.org", then, even without the space, the syntax here is backwards: that would be `[Node.js](https://nodejs.org)`.